rialto_airflow/database.py

```diff
@@ -69,7 +69,7 @@ def pg_utcnow(element, compiler, **kw):
 )
 
 
-class Publication(Base):
+class Publication(Base):  # type: ignore
```

Comment:
there's some magic used in `Base` that trips up mypy.  if/when airflow becomes sqlalchemy 2.x compatible, this should no loner be needed, because sqlalchemy 2.x is supposed to play better with type checkers.  (i think sqlalchemy 1.x predates widespread typechcker usage?)

Reply:
That sounds plausible to me.